src/neomaril_codex/model.py

```diff
@@ -79,14 +107,33 @@ def __get_status(self):
             raise ModelError(response.text)
 
     def wait_ready(self):
+        """
+        Waits the model to be with status 'Ready'
+        
+        Example
+        -------
+        >>> model.wait_ready()
```

Comment:
I think we can improve those exemples. For this case it only make sense being used after getting a model that is not ready, so we can show in the example a model and get its status and then using this method to wait and then checking if its really ready to use. If we cant think in a complete example maybe just the documentation is enough.
Try to think if the method needs:
- Some method is only need in a specific scenario, so show a explanation to when we need to use the method
- Multiple parameters so show when/how to use each of them